tools/java/org/unicode/cldr/util/TempPrintWriter.java

```diff
@@ -22,6 +22,7 @@
     final PrintWriter tempPrintWriter;
     final String tempName;
     final String filename;
+    boolean noReplace = false;
 
 
     public static TempPrintWriter openUTF8Writer(String filename) {
```

Comment:
Modified this so that the temp files can be removed if there is an exception. Avoids muddying the directories.